src/components/WorkPoolQueuesTable.vue

```diff
@@ -110,7 +108,7 @@
 
   const handleDelete = async (): Promise<void> => {
     await workPoolQueuesSubscription.refresh()
-    selected.value = selected.value?.filter(queue => workPoolQueues.value.find(({ id }) => id === queue.id))
+    selected.value = []
```

Comment:
just deselect everything on successful deletion. that's what all the other tables do